doc/contributor/howto-guide-declare-a-library-ga.md

```diff
@@ -2,96 +2,114 @@
 
 This document describes the steps required to promote a `google-cloud-cpp`
 library to GA. The document is intended for contributors to the
-`google-cloud-cpp` libraries. It assumes you are familiar with the build systems
-used in these libraries, and that you are familiar with existing libraries.
+`google-cloud-cpp` libraries. It assumes you are familiar with the build systems used in these libraries, and that you
+are familiar with existing libraries.
```

Comment:
It was a little difficult to follow the diff because of these other reflows.  Do we even want them given they now exceed our normal line-length limitation for text?

Reply:
Fixed. #8470 is starting to look more tempting.